### PR TITLE
Show update message when wallet version outdated

### PR DESCRIPTION
**Pull request by nop33**
Related to #141

https://user-images.githubusercontent.com/1579899/160446686-3161098c-db15-4df5-8b3c-df6683924cec.mp4

### CODE REVIEW

**Comment by nop33** [comment]
I didn't implement a permanent banner yet as @mvaivre requested, I can work on that tomorrow.

**Comment by lf94** [comment]
Looks good!

**Comment by mvaivre** [comment]
Looks good indeed ! Here's what I had in mind regarding the banner :

image

**Comment by nop33** [comment]
Sweet! On it!

**Comment by nop33** [comment]
Finished! Ready for review

**Review by mvaivre — COMMENTED** [review]
Two little comments! Otherwise, LGTM.

File: src/hooks/useLatestGitHubRelease.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useEffect, useState } from 'react'
+
+import { useGlobalContext } from '../contexts/global'
+
+const currentVersion = process.env.REACT_APP_VERSION
+const semverRegex = /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)?$/
```
[the remark this answers is not in the document]
Comment (nop33) [reply src/hooks/useLatestGitHubRelease.ts:24]: I have tested it here: https://regex101.com/r/C8h69Q/1

I looked quickly into the semver package, but I didn't find an easy way to check if the version is of the format X.Y.Z without anything else (which is what the regex above does).

File: src/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export default styled.div`
+  background-color: ${({ theme }) => theme.global.accent};
+  color: var(--color-white);
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1001;
+  position: fixed;
```
[the remark this answers is not in the document]
Comment (nop33) [reply src/components/Banner.tsx]: fixed!

**Review by mvaivre — APPROVED** [review]

**Pull request merged**